src/CustomerioTracking.tsx

```diff
@@ -143,6 +143,35 @@ class CustomerIO {
     }
     CustomerioReactnative.registerDeviceToken(token)
   }
+
+  /**
+   * Request to show prompt for push notification permissions.
+   * Prompt will only be shown if the current status is - not determined.
+   * In other cases, this function will return current status of permission.
+   * @param options
+   * @returns Success & Failure promises
+   */
+  static async showPromptForPushNotifications(options?: any) : Promise<any>{
+    let pushConfigurationOptions  = {
+      "ios": {
+        "badge" : true,
+        "sound" : true
+      }
+    }
+    if (typeof options !== 'undefined'){
+      pushConfigurationOptions = options
+    }
+  
+  return CustomerioReactnative.showPromptForPushNotifications(pushConfigurationOptions)
+  }
+
+  /**
+   * Get status of push permission for the app
+   * @returns Promise with status of push permission as a string
+   */
+  static getPushPermissionStatus() : Promise<any> {
```

Comment:
```suggestion
  static getPushPermissionStatus() : Promise<PushPermissionStatus> {
```

Going along with the comment about a strongly typed parameter for `showPromptForPushNotifications`, we know the data type that is being returned from the `Promise`. 

We should be providing customers with a data type return value that they can easily parse in their code.